backend/bufftracer/src/tagSpanTrace.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  APIRequestMetadata,
+  GraphQlRequest,
+  isGraphQlRequest,
+  isRPCRequest,
+  RPCRequest,
+  TAG_NAME,
+} from './types'
+import { parseRPCRequest } from './parseRPCRequest'
+import { parseGraphQLRequest } from './parseGraphQLRequest'
+import { RESOURCE_NAME } from 'dd-trace/ext/tags'
+import { Span } from 'dd-trace'
+
+export function tagSpanTrace(
+  span: Span | undefined,
+  req: RPCRequest | GraphQlRequest,
+  errorCallback: (ex: Error) => void,
+): boolean {
```

Comment:
Not sure if it's needed to return anything here 🤔  Looks like it's not used where it's called. Maybe `void` would be more accurate?
I see that you use it in tests, but maybe we can change that and check if `errorCallback` was called or not 😇 .

Reply:
Great catch, indeed I started returning boolean for the unit tests, but I don't like it! I will change that